spec/services/cfe/proceedings_payload_service_spec.rb

```diff
@@ -9,7 +9,8 @@
       let(:session_data) do
         {
           "level_of_help" => "certificated",
-          "matter_type" => "other",
+          "domestic_abuse_applicant" => "false",
```

Comment:
This should strictly be `"domestic_abuse_applicant" => false,` (no quotation marks around the value).